content/posts/2023/review-africa-addio.md

```diff
@@ -0,0 +1,45 @@
++++ 
+date = 2023-02-26T12:08:18Z
+title = "Review: Africa Addio (1966)"
+tags = ["review", "movie"]
+categories = ["life"]
++++
+
+I have an extensive list of movies to watch that sparked my interest somehow.
+I don't keep notes on why I wanted to watch them, or where I first learned about them.
+Most often enough time passes that when I finally get to watch it, I have no idea what's about.
+It ends up being a good strategy, since I get to see it with completely fresh eyes.
+
+This week I got to watch Africa Addio (1996), or Africa: Blood and Guts.
+
+
+Africa Addio (1966) is an Italian documentary about the effects of the end of the
+European Colonial Era in Africa. Right at the beginning it sets its tone, saying that **"Europe has abandoned its baby when it needed the most"** or something like that, reverberating that adage that **Europe made 2 mistakes in Africa: colonizing it, and then leaving it behind.**
+
+Technically it's a pretty good movie, with a superb artsy photography that get **their** point across.
+
+
+There's one part where the English (I think) are selling their stuff to the new
+African elites, where the narrator implies that they don't know what to do with it, since
+the Africans are just mimicking the European elites. Which reminded me of Rene Girard's mimetic desire theory.
```

Comment:
```suggestion
African elites, where the narrator implies that the Africans don't know what to do with it, since
they are just mimicking the European elites. Which reminded me of Rene Girard's mimetic desire theory.
```